resources/conf/config.toml.template

```diff
@@ -119,6 +119,13 @@
   # Disable SSL verification
   disableSslVerification = false
 
+# Configure timeout settings related to routes. This will be applicable globally for all the envoy routes.
+[router.timeouts]
+# Specifies the upstream timeout for the route. If not specified, the default is 30s.
+backendTimeoutinSeconds = 30
+# Backend connection idle timeout. The amount of time the request’s stream may be idle.
+idleTimeoutinSeconds = 300
+
```

Comment:
Recheck the configs that need to be added to the template file.

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2064/commits/9f3de335c1938ad0889a7f181ccb0e2504672d0b